lib/command.ts

```diff
@@ -171,4 +171,5 @@ export default abstract class BalenaCommand extends Command {
 
 	protected outputMessage = output.outputMessage;
 	protected outputData = output.outputData;
+	protected printTitle = output.printTitle;
```

Comment:
This allows commands to output a title explicitly (in the same format we get in the tabular data).